http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -224,8 +224,13 @@ private[leonardo] object LeoProfile extends MySQLProfile {
       MappedColumnType.base[AppId, Long](_.id, AppId.apply)
     implicit val appNameColumnType: BaseColumnType[AppName] =
       MappedColumnType.base[AppName, String](_.value, AppName.apply)
+    implicit val appAccessScopeColumnType: BaseColumnType[AppAccessScope] =
+      MappedColumnType.base[AppAccessScope, String](
+        _.toString,
+        s => AppAccessScope.stringToObject.getOrElse(s, throw ColumnDecodingException(s"invalid app access scope ${s}"))
+      )
     implicit val appSamResourceIdColumnType: BaseColumnType[AppSamResourceId] =
-      MappedColumnType.base[AppSamResourceId, String](_.resourceId, AppSamResourceId.apply)
+      MappedColumnType.base[AppSamResourceId, String](_.resourceId, s => AppSamResourceId.apply(s, None))
```

Comment:
Hm, is it an issue that we're deserializing `AppAccessScope` to `None` here? I guess maybe this field isn't used in the DB layer?